chord_metadata_service/chord/ingest/phenopackets.py

```diff
@@ -264,8 +264,12 @@ def get_or_create_genomic_interpretation(gen_interp: dict) -> pm.GenomicInterpre
 def get_or_create_disease(disease) -> pm.Disease:
     d_obj, _ = pm.Disease.objects.get_or_create(
         term=disease["term"],
+        excluded=disease.get("excluded"),
```

Comment:
per phenopackets v2 spec, if this isn't present it defaults to "false" - so should this be `get("excluded", False)` and changed in the model to be non-null + default False?